front-end/src/utils/DropdownStyles.js

```diff
@@ -0,0 +1,36 @@
+export const customStyles = {
+    menu: (provided) => ({ 
+        ...provided,
+        background: '#b3b3b3',
+    }),
+    option: (provided, state) => ({ 
+        ...provided,
+        background: state.isFocused ? '#acf39d' : '#b3b3b3',
+            '&:active': {
+                background: '#3cdd1b'
+            },
+        color: state.isFocused ? '#f9f9f8' : '#2d2d2d',  
```

Comment:
If possible, could you maybe make the highlighted option text color a bit darker? I find it's a bit hard to read with the light green background.

Other than that it looks good!